dev-python/pypubsub/pypubsub-4.0.3.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{4,5,6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="A Python publish-subcribe library"
+HOMEPAGE="https://github.com/schollii/pypubsub/"
+SRC_URI="
+	https://github.com/schollii/pypubsub/archive/v${PV}.tar.gz -> ${P}.tar.gz
```

Comment:
Why make it a 3-liner? :)

Reply:
Because it's cleaner to not have the delimiters on the same line as soon as there's more than one.